src/index.css

```diff
@@ -1,5 +1,108 @@
 body {
   margin: 0;
   padding: 0;
-  font-family: sans-serif;
+  font-family: Arial, sans-serif;
+}
+
+.stock-tracker {
+  width: 1200px;
+  margin: 20px;
+  padding: 20px;
+  background-image: linear-gradient(127deg, #012861, #021738 69%, #021025);
+  border-radius: 5px;
+}
+
+.search {
+  height: 50px;
+  width: 100%;
+  background: transparent;
+  border: none;
+  border-bottom: solid 2px #7fb3ff;
+  color: white;
+  font-size: 30px;
+  outline: none;
+}
+
+.result-list {
+  list-style-type: none;
+  color: #beccdc;
```

Comment:
You can use variable in CSS.
[https://developer.mozilla.org/en-US/docs/Web/CSS/:root](url)